ckanext/datavic_iar_theme/helpers.py

```diff
@@ -110,3 +110,18 @@ def visibility_list():
         {"value": "private", "label": "Open to VPS only"},
         {"value": "public", "label": "Open to the public"}
     ]
+
+def featured_resource_preview(resources):
+    featured_preview = None
+    for item in resources:
+        if item.get('featured') is True:
```

Comment:
Be more idiomatic: `if item.get('featured'):` has the same sense. If `featured` is a boolean value all the time, of course